CASMPET-5557 1.0 : Add Set_BMC_Management_Role to Add/Remove/Replace NCN procedure (#1431)
Cray-HPE/docs-csm: operations/node_management/Add_Remove_Replace_NCNs.md

All activities required for site maintenance are complete.

> The latest docs-csm RPM has been installed on the master nodes. See [Update CSM Product Stream](../../update_product_stream/index.md) for information in installing the latest docs-csm documentation RPM.

Reviewer: ```suggestion
The latest `docs-csm` RPM has been installed on the master nodes. See [Update CSM Product Stream](../../update_product_stream/index.md) for more information.
```
```suggestion
The latest docs-csm RPM has been installed on the master nodes. See [Update CSM Product Stream](../../update_product_stream/index.md) for information in installing the latest docs-csm documentation RPM.
```
